ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts

```diff
@@ -0,0 +1,19 @@
+import { callbacks } from '../../../../../app/callbacks/server';
+import { LivechatDepartment } from '../../../../../app/models/server';
+import { settings } from '../../../../../app/settings/server';
+
+callbacks.add('livechat.applySimultaneousChatRestrictions', ({ departmentId }: { departmentId?: string }) => {
+	if (departmentId) {
+		const departmentLimit = departmentId ? LivechatDepartment.findOneById(departmentId)?.maxNumberSimultaneousChat : 0;
```

Comment:
```suggestion
		const departmentLimit = LivechatDepartment.findOneById(departmentId)?.maxNumberSimultaneousChat || 0;
```

Reply:
To fix this LGTM bot error: https://lgtm.com/projects/g/RocketChat/Rocket.Chat/snapshot/c00e13b80bd4b7af5e17ba648575b044e4eb852f/files/ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts?sort=name&dir=ASC&mode=heatmap

Reply:
Thanks!